packages/cli/src/Initializer/index.ts

```diff
@@ -396,6 +397,23 @@ export default class Initializer {
     return false
   }
 
+  static async isFlareact(package_json: PackageJson) {
+    const expo_dep =
+      package_json.dependencies?.['flareact'] ||
+      package_json.devDependencies?.['static_dir']
```

Comment:
@geelen what is the reason for checking for this devDependency? Is it inside the equation?

Reply:
Lol that's a mistake. Should be `'flareact'` in both cases